src/support_sphere/lib/presentation/components/auth/signup_form.dart

```diff
@@ -91,6 +72,7 @@ class _FirstNameInput extends StatelessWidget {
             ],
             value,
             context,
+            context.read<SignupCubit>(),
```

Comment:
The context is passed into the function, I'm wondering what is the reasoning of passing in the Cubit like this rather than just a simple `SignupCubit` and then calling the `context.read<Cubit>()` within the function?

Reply:
I think it is better to pass the class ~ more reusable. 
I didn't notice it much when I made the commit previously, just checked how we access the cubit and had that solution.